gallery/package.json

```diff
@@ -51,6 +51,10 @@
     "jest-image-snapshot": "^4.5.1",
     "jest-junit": "^12.3.0",
     "mini-css-extract-plugin": "^2.6.0",
+    "@nivo/core": "^0.75.0",
+    "@nivo/bar": "^0.79.1",
+    "@nivo/line": "^0.75.0",
+    "@nivo/pie": "^0.78.0",
```

Comment:
0.80.0 is the current latest version of all of these.  I don't see a reason they shouldn't all be on the same version

Reply:
> 0.80.0 is the current latest version of all of these. I don't see a reason they shouldn't all be on the same version

Updated here: 48acb84c61af8550c312fcba44c66372d4dc1246